src/liboslexec/llvm_gen.cpp

```diff
@@ -1516,7 +1516,7 @@ LLVMGEN (llvm_gen_construct_triple)
         llvm::Value *args[] = { rop.sg_void_ptr(),
             rop.llvm_void_ptr(Result), rop.ll.constant(Result.has_derivs()),
             rop.llvm_void_ptr(Result), rop.ll.constant(Result.has_derivs()),
-            rop.llvm_load_value(Space), rop.ll.constant(Strings::common),
+            rop.llvm_load_string(Space), rop.ll.constant(Strings::common),,
```

Comment:
Maybe in the long run, it's smart to make llvm_load_value "string-aware" so that we don't have to change all these (including in future PRs covering other parts of the code)? Also, do you have an extra comma on this line?

Reply:
I like that idea, I'll look into it. Yup, you're correct that comma got away from me. Fixed in dc976f7.

Reply:
I've made `llvm_load_string` "string-aware" in 5fc8b1a and reverted the modifications in `llvm_gen.cpp`. I'm not sure how robust the ternary condition is but it seems to work correctly in my testing.